edgar-doctor/src/components/app/dashboardPages/appointments/AppointmentsCards.tsx

```diff
@@ -15,8 +15,16 @@ import { type AppointmentType } from 'types/app/dashboard/appointments/appointme
 const AppointmentsCards = ({ appointments }: { appointments: AppointmentType[] }): JSX.Element => {
 	const [pageIndexNext, setPageIndexNext] = useState(1);
 	const [pageIndexOld, setPageIndexOld] = useState(1);
-	const upcomingAppointments = appointments.filter((appointment) => appointment.endDate > new Date().getTime());
-	const pastAppointments = appointments.filter((appointment) => appointment.endDate <= new Date().getTime());
+	const upcomingAppointments = appointments.filter(
+		(appointment) =>
+			appointment.endDate > new Date().getTime() &&
+			!['CANCELED', 'CANCELED_DUE_TO_REVIEW'].includes(appointment.appointmentStatus),
+	);
+	const pastAppointments = appointments.filter(
+		(appointment) =>
+			appointment.endDate <= new Date().getTime() &&
+			!['CANCELED', 'CANCELED_DUE_TO_REVIEW'].includes(appointment.appointmentStatus),
```

Comment:
A bit complex, I think that `appointment.appointmentStatus.includes('CANCELED')` is enough for this case because we don't want to display any canceled appointment